updater/lib/tinglesoftware/azure/artifacts_credential_provider.rb

```diff
@@ -0,0 +1,48 @@
+# typed: strict
+# frozen_string_literal: true
+
+require "dependabot/shared_helpers"
+
+#
+# This module auto installs the Azure Artifacts Credential Provider if any Azure DevOps NuGet feeds are configured.
+# Without it, users have to deal with complicated workarounds to get authentication to work with Dependabot updates.
+# See: https://github.com/tinglesoftware/dependabot-azure-devops/pull/1233 for more info.
+#
+
+# TODO: Remove this once https://github.com/dependabot/dependabot-core/pull/8927 is resolved or auth works natively.
+
+module TingleSoftware
+  module Azure
+    module ArtifactsCredentialProvider
+      def self.install_if_nuget_feeds_are_configured
```

Comment:
Where is this invoked?

Reply:
On line 49, very bottom of file; It self-invokes when the script is included using `requires`.

Reply:
Seen it. 👍🏾